cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -161,20 +162,19 @@ func runDescribeCluster(name string) error {
 // printObjectTree prints the cluster status to stdout.
 func printObjectTree(tree *tree.ObjectTree) {
 	// Creates the output table
-	tbl := uitable.New()
-	tbl.Separator = "  "
-	tbl.AddRow("NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE")
+	tbl := tablewriter.NewWriter(os.Stdout)
+	tbl.SetHeader([]string{"NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE"})
```

Comment:
```suggestion
	tbl.SetHeader([]string{"NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE"})
	tbl.SetAutoWrapText(false)
	tbl.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
	tbl.SetAlignment(tablewriter.ALIGN_LEFT)
	tbl.SetCenterSeparator("")
	tbl.SetColumnSeparator("")
	tbl.SetRowSeparator("")
	tbl.SetHeaderLine(false)
	tbl.SetBorder(false)
	tbl.SetTablePadding("  ")
	tbl.SetNoWhiteSpace(true)
```
To match the output format of the new table to the old table we will need these rules. These formatting rules will guarantee that the output does not change with the new table library.

Reply:
yes done thanks for the input